at_common_flutter/lib/widgets/custom_toast.dart

```diff
@@ -0,0 +1,49 @@
+
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+
+class CustomToast {
+  /// pass [isError] true to have a red bg, [isSuccess] true to have a green bg.
+  static void show(String message, BuildContext? context,
+          {Color? bgColor,
+          Color? textColor,
+          double? toastWidth,
```

Comment:
Can we add default value to toastWidth ?

Reply:
This field is not so important. So we made it a null value without any default value to it.